src/microbe_stage/MicrobeCamera.cs

```diff
@@ -94,6 +77,11 @@ public partial class MicrobeCamera : Camera3D, IGodotEarlyNodeResolve, ISaveLoad
     [JsonProperty]
     private float lightLevel = 1.0f;
 
+#pragma warning disable CA2213
+    [JsonIgnore]
+    private BackgroundPlane backgroundPlane = null!;
```

Comment:
I prefer putting the Godot variables before the "normal" variables so I'd prefer if this bit was kept around line 65 like where ` private Node3D? backgroundPlane;` was previously.